### PR TITLE
DSD-1746: FeedbackBox interaction tests

### PR DESCRIPTION
**Pull request by 7emansell**
Fixes JIRA ticket [DSD-1746](https://newyorkpubliclibrary.atlassian.net/browse/DSD-1746)

## This PR does the following:

- Adds interaction tests in Storybook for the `FeedbackBox` component

### Checklist:

- [x] I have updated the Storybook documentation accordingly.
- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

[DSD-1746]: https://newyorkpubliclibrary.atlassian.net/browse/DSD-1746?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/FeedbackBox/FeedbackBox.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Box } from "@chakra-ui/react";
 import type { Meta, StoryObj } from "@storybook/react";
 import { useState } from "react";
+import { userEvent, expect, screen } from "@storybook/test";
```
Comment (EdwinGuzman) [inline src/components/FeedbackBox/FeedbackBox.stories.tsx:4]: Did not know you can get the mounted DOM directly from the package. I've been doing `const screen = within(canvasElement);`

**Review by EdwinGuzman — APPROVED** [review]
It'd be nice to get tests for the other fields but I'm not sure if you can rerender the component and set `showCategoryField` and `showEmailField` to true. Maybe those should be set to true for storybook, but that can be separate and a later step.

File: src/components/FeedbackBox/FeedbackBox.stories.tsx (hunk 1 above)
Comment (7emansell) [reply src/components/FeedbackBox/FeedbackBox.stories.tsx:4]: Right, something to remember for `Modal` for sure bc `screen` here is not the same as `within(canvasElement)`

**Comment by 7emansell** [comment]
> It'd be nice to get tests for the other fields but I'm not sure if you can rerender the component and set `showCategoryField` and `showEmailField` to true. Maybe those should be set to true for storybook, but that can be separate and a later step.

@bigfishdesign13 I think we've done this before with the `Searchbar` interaction tests (added props to the default component just so they can be tested) so I think it would make sense to do here as well, but lmk

**Comment by bigfishdesign13** [comment]
> > It'd be nice to get tests for the other fields but I'm not sure if you can rerender the component and set `showCategoryField` and `showEmailField` to true. Maybe those should be set to true for storybook, but that can be separate and a later step.
> 
> @bigfishdesign13 I think we've done this before with the `Searchbar` interaction tests (added props to the default component just so they can be tested) so I think it would make sense to do here as well, but lmk

@7emansell Yes, I think that would be fine to set `showCategoryField` and `showEmailField` to `true` for the `With Controls` story. 

@EdwinGuzman The next release is next Thursday, so there would be time to tackle this change as part of this ticket.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**